note_seq/alignment/align_fine_lib.py

```diff
@@ -129,14 +129,14 @@ def align_cpp(samples,
   # Write to file.
   fh, temp_path = tempfile.mkstemp(suffix='.proto')
   os.close(fh)
-  with open(temp_path, 'w') as f:
+  with open(temp_path, 'wb') as f:
     f.write(alignment_task.SerializeToString())
 
   # Align with C++ program.
   subprocess.check_call([ALIGN_BINARY, temp_path])
 
   # Read file.
-  with open(temp_path + '.result') as f:
+  with open(temp_path + '.result', "rb") as f:
```

Comment:
we typically use single quotes. can you switch to be consistent?

Reply:
Here you are :)